src/pages/index.vue

```diff
@@ -1,27 +1,88 @@
-                rounded="xl"
-                @click="selectedIndicator = name"
-                >{{ name }}
-            </v-btn>
+    <template v-if="!unknownIndicator && !unknownCountry">
+        <Choropleth data-testid="choropleth" />
+        <div class="sticky-footer">
+            <div v-for="name in indicatorNames" :key="name">
+                <router-link :to="`/${name}/${selectedCountryId}`" custom v-slot="{ navigate }">
+                    <v-btn
+                        @click="navigate"
+                        role="link"
+                        class="mb-2"
+                        :class="name === selectedIndicator ? 'bg-blue' : 'bg-black'"
+                        rounded="xl"
+                        >{{ name }}
+                    </v-btn>
+                </router-link>
+            </div>
         </div>
-    </div>
+    </template>
+    <not-found v-else :detail="notFoundDetail"></not-found>
 </template>
 
 <script lang="ts" setup>
 import { storeToRefs } from "pinia";
-import { computed } from "vue";
+import { computed, Ref, watch, ref } from "vue";
+import { useRouter } from "vue-router";
 import { useAppStore } from "../stores/appStore";
+import NotFound from "./notFound.vue";
 
-const { appConfig, selectedIndicator } = storeToRefs(useAppStore());
+const router = useRouter();
+const { appConfig, selectedIndicator, selectedCountryId } = storeToRefs(useAppStore());
+const { selectCountry } = useAppStore();
+
+const props = defineProps({
+    indicator: {
+        type: String,
+        required: false,
+        default: ""
+    },
+    country: {
+        type: String,
+        required: false,
+        default: ""
+    }
+});
 
 const indicatorNames = computed(() => (appConfig.value ? Object.keys(appConfig.value.indicators) : {}));
+const unknownCountry: Ref<boolean> = ref(false);
+const unknownIndicator: Ref<boolean> = ref(false);
+
+const notFoundDetail = computed(() => {
+    return (
+        (unknownIndicator.value ? `Unknown indicator: ${props.indicator}. ` : "") +
+        (unknownCountry.value ? `Unknown country ISO: ${props.country}.` : "")
+    );
+});
+
+const selectDataForRoute = async () => {
+    if (!appConfig.value) {
+        return;
+    }
+
+    if (appConfig.value) {
+        if (props.indicator) {
+            // Do case-insensitive match to find indicator from route
+            const pattern = new RegExp(`^${props.indicator}$`, "i");
+            const indicator = Object.keys(appConfig.value.indicators).find((i) => pattern.test(i));
+            unknownIndicator.value = !indicator;
+
+            const country = props.country ? props.country.toUpperCase() : "";
+            unknownCountry.value = country && !appConfig.value.countries.includes(country);
+
+            if (!unknownIndicator.value && !unknownCountry.value) {
+                selectedIndicator.value = indicator;
+                await selectCountry(country);
```

Comment:
just noticed but we should try {} catch {} our promises because they can be rejected... that should apply for all cases being drilled down

Reply:
Yeah, though something's gone badly wrong if that's the case. Will add a separte ticket for error handling.

Reply:
https://mrc-ide.myjetbrains.com/youtrack/issue/mrc-5273/Error-handling